tests/www/api/experimental/test_dag_runs_endpoint.py

```diff
@@ -35,6 +35,9 @@ def setUpClass(cls):
         session.query(DagRun).delete()
         session.commit()
         session.close()
+        dagbag = DagBag(include_examples=True)
```

Comment:
I don't understand this change, I assume it was necessary to load the example DAGs? Did it not do that before this change?

Reply:
Actually there is a need for this. In `test_jobs.py`, we clear the `dag` table before the tests, which includes the example dags used by other tests such as `test_dag_runs_endpoint.py`. In the past `test_jobs.py` runs after all other tests so everything looks fine.

However, after this change, the test execution order also changes and it expose the race condition here. So I added this change to reload the `dag` table in tests that requires example dags in the `dag` table.

Reply:
Clear, thanks!